docs/runtime_options.md

```diff
@@ -158,23 +158,124 @@ automatically.
 * `grouping` - in scenario when accounted for events are used to calculate derivative metrics, it is reasonable to
 measure them in transactional manner: all the events in a group must be accounted for in the same period of time. Keep
 in mind that it is impossible to group more events that there are counters available.
+* `uncore events` - events which can be counted by PMUs outside core.
+* `PMU` - Performance Monitoring Unit
+
+#### Getting config values
+Using perf tools:
+* Identify the event in `perf list` output. 
+* Execute command: `perf stat -I 5000 -vvv -e EVENT_NAME`
+* Find `perf_event_attr` section on `perf stat` output, copy config and type field to configuration file.
+
+```
+------------------------------------------------------------
+perf_event_attr:
+  type                             18
+  size                             112
+  config                           0x304
+  sample_type                      IDENTIFIER
+  read_format                      TOTAL_TIME_ENABLED|TOTAL_TIME_RUNNING
+  disabled                         1
+  inherit                          1
+  exclude_guest                    1
+------------------------------------------------------------
+```
+* Configuration file should look like: 
+```json
+{
+  "core": {
+    "events": [
+      ["EVENT_NAME"]
+    ],
+    "custom_events": [
+      {
+        "type": 18,
+        "config": [
+          "0x304"
+        ],
+        "name": "EVENT_NAME"
+      }
+    ]
+  },
+  "uncore": {
+    "events": [
+      ["EVENT_NAME"]
+    ],
+    "custom_events": [
+      {
+        "type": 18,
+        "config": [
+          "0x304"
+        ],
+        "name": "EVENT_NAME"
+      }
+    ]
+  }
+}
+```
+
+Config values can be also obtain from: 
+* [Intel® 64 and IA32 Architectures Performance Monitoring Events](https://software.intel.com/content/www/us/en/develop/download/intel-64-and-ia32-architectures-performance-monitoring-events.html)
+
+
+##### Uncore Events configuration
+Uncore Event name should be in form `PMU_PREFIX/event_name` where **PMU_PREFIX** mean
```

Comment:
apologies if this has already been discussed, but why do we need to be able to provide a prefix for the PMU?

Reply:
Is there always an integer `type`, or do we sometimes need the string prefix?  In what cases can't we use the integer type?

Reply:
Let me explain this. 
If you don't pass `type` in custom events, PMU will be taken from `PMU_PREFIX`.
`PMU_PREFIX` is the name of PMU.

Reply:
We don't need both ways. Because setting type field in custom events was before uncore events, I added that possibility.
I decided also to add `perf stat` tool way of handling PMU ( **PMU_PREFIX/event_name** ).

Reply:
I've had some time to dig into this.  Thanks to @iwankgb for explaining some things to me.  I'm not a fan of having "/" -delineated strings in our config, and generally prefer explicit fields for each input, but also understand that this is used elsewhere.  I'll let you decide what to do.